allRoutes/bearRoutes.js

```diff
@@ -0,0 +1,103 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+const db = knex(knexConfig.development);
+console.log(knexConfig);
+const router = express.Router();
+
+// endpoints here
+//++++++++++++++++++++++++++++++++++++++++
+// - post stuff here
+//++++++++++++++++++++++++++++++++++++++++
+router.post('/', (req, res) => {
+    console.log('request working');
+    const content = req.body;
+    db('daBears')
+        .insert(content)
+        .into('daBears')
+        .then( ids => {
+            res.status(200).json(ids);
+        })
+        .catch(err => {
+            res.status(500).json({error: "error", err});
+        });
+});
+
+
+//++++++++++++++++++++++++++++++++++++++++++
+// get endpoints
+//++++++++++++++++++++++++++++++++++++++++++++
+router.get('/', (req, res) => {
```

Comment:
Another option for routing with express is using the `route` method, which enables you to chain endpoints:
```javascript
router.route('/')
  .get((req, res) => { /* GET all callback logic */ })
  .post((req, res) => { /* POST callback logic */ });

router.route(':/id')
  .get((req, res) => { /* GET by id callback logic */ })
  .delete((req, res) => { /* DELETE callback logic */ })
  .put((req, res) => { /* PUT callback logic */ });
```

Reply:
People typically will use 2 (or more) files for this, one(+) for the routes and one(+) for the callback functions themselves.